data_architect_misc/data_transformer/transform_functions/apac_singapore_transform_functions.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, config):
         # specific category mapping for this country
         self.category_mappings = comp_harm_constants.ENGLISH_CATEGORY_MAPPINGS
 
-    def add_RAW_BRAND_column_for_multiple_files_using_specific_string_part_of_filename(
+    def create_new_dataframe_and_add_RAW_BRAND_column_for_multiple_files_using_specific_string_part_of_filename(
```

Comment:
Please rename this method to: "create_new_dataframe_with_RAW_BRAND_column_from_multiple_input_file_with_brand_names_as_part_of_the_file_names"

Reply:
change done